### ISSUE TITLE
Data structure in this repo - discussion

### ISSUE THREAD

**Issue by kat-kap** [acism/awesome-scrum-master#10]
This issue is to discuss how we want to structure data and ensure transparency in this repo in future.

**Comment by fadehelix** [comment]
First of all "Scrum" can be splitted into smaller subjects like "Planning", "Refinement", "Retrospective", "Daily" etc.
Kanban is ok for me,
DevOps - i can't see how it can be related to the Scrum @robertjanista  ;)
I am not sure about "Working with people" and "Working with the team" - isn't it something like "Facilitation", or "Team building" ?

**Comment by kat-kap** [comment]
My name is not Robert, but I like the idea of having devops section here. Imho scrum masters should understand devops concept and culture. It is tied closely with the development process and shipping value to the customer, and SMs should definitely know how engineers can become more fast and agile when it comes to delivering and releasing. Or what happens when we lack devops mindset and processes :)

**Comment by robertjanista** [comment]
Yeah, at first having stuff like DevOps might seem excessive, but actually DevOps is based on Agile and Lean, and I think it's a useful knowledge to any leadership position in the world of technology.

I would argue that we could have "Software development" category for stuff like books on Software delivery lifecycle and good practices (think of "Extreme programming explained" or "Lean software development") - that might be also handy for Scrum Masters.

But I might be biased since I have background in tech and I still do things like devops coaching e.g. during implementation of our ci/cd model (delivery, not deployment yet) which was started as my inititative.

**Comment by fadehelix** [comment]
Ok, it makes sense. Do you know any article where someone explains which technical knowledge/skills are useful for "average" Scrum Master? IMO would be nice to provide some context for subjects like DevOps,  programming fundamentals, etc.

**Comment by robertjanista** [comment]
Unfortunately still struggling with finding good books/articles about programming for non-tech people.

**Comment by fadehelix** [comment]
@robertjanista so maybe that's the challenge for me and you? :) As we  have "some" experience in both programming and non-technical areas ? :) I am going to think about topics. We can also ask folks from SACiSM what they would expect to explain.

**Comment by kat-kap** [comment]
I have a huge list of books about programming useful to SMs :) Everyone should read clean code. I'll do my best to upload it during weekend, hope I'll manage :)

**Comment by kat-kap** [comment]
You can review it and we will remove if you don't agree with something on that list. But basically it's about getting familiar with every tech area that the team is involved with. Programing, testing, releasing, maintenance, full sdlc imo :)

**Comment by kat-kap** [comment]
But this discussion was supposed to be about the data structure xD I will create a separate issue :)
